passes/techmap/abc9.cc

```diff
@@ -952,6 +952,42 @@ struct Abc9Pass : public Pass {
 #endif
 #endif
 
+		// get arguments from scratchpad first, then override by command arguments
+		std::string lut_arg, luts_arg;
+		if (design->scratchpad.count("abc9.script")) {
+			script_file = design->scratchpad_get_string("abc9.script");
+		}
+		if (design->scratchpad.count("abc9.D")) {
+			delay_target = "-D " + design->scratchpad_get_string("abc9.D");
+		}
+		if (design->scratchpad.count("abc9.lut")) {
+			lut_arg = design->scratchpad_get_string("abc9.lut");
+		}
+		if (design->scratchpad.count("abc9.luts")) {
+			luts_arg = design->scratchpad_get_string("abc9.luts");
+		}
+		if (design->scratchpad.count("abc9.fast")) {
+			fast_mode = design->scratchpad_get_bool("abc9.fast", false /* default value if not bool-like */);
```

Comment:
How about `design->scratchpad_get_bool("abc9.fast", fast_mode)` so that it inherits the default value?